### PR TITLE
Added User.GetUsersStatusesByIds

### PR DESCRIPTION
**Pull request by ethervoid**
Added the method User.GetUsersStatusesByIds for the user entity

https://mattermost.atlassian.net/browse/MM-20844

### CODE REVIEW

File: loadtest/user/userentity/actions.go [hunk 1]
```diff
@@ -127,3 +127,8 @@ func (ue *UserEntity) GetChannelUnread(channelId string) (*model.ChannelUnread,
 	return channelUnreadResponse, resp.Error
 
 }
+
+func (ue *UserEntity) GetUsersStatusesByIds(userIds []string) ([]*model.Status, error) {
```
Comment (ethervoid) [inline loadtest/user/userentity/actions.go]: Is there a point to check here for user initialization like [here](https://github.com/mattermost/mattermost-load-test-ng/pull/17/files#diff-b68f89ec953f6d29ef590768f6d33f9eR124) for example? I understand that this should be done if we're going to use the user object but here I don't see the point

**Comment by reflog** [comment]
@ethervoid - please resolve the conflicts
@streamer45 - does this method needs to update the store?

**Comment by streamer45** [comment]
> @ethervoid - please resolve the conflicts
> @streamer45 - does this method needs to update the store?

We do store statuses on webapp and at some point we might as well here. But for now I don't see an immediate reason to store that data especially since we are yet not handling websocket updates.

File: loadtest/user/user.go [hunk 2]
```diff
@@ -36,4 +36,7 @@ type User interface {
 	GetChannelStats(channelId string) error
 
 	// teams
+
+	// users
+	GetUsersStatusesByIds(userIds []string) ([]*model.Status, error)
```
Comment (streamer45) [inline loadtest/user/user.go]: Let's just return an error for now.

Comment (ethervoid) [reply loadtest/user/user.go]: Done

File: example/sampleuser/user.go [hunk 3]
```diff
@@ -90,6 +90,10 @@ func (u *SampleUser) GetChannelStats(channelId string) error {
 	return nil
 }
 
+func (u *SampleUser) GetUsersStatusesByIds(userIds []string) ([]*model.Status, error) {
```
Comment (streamer45) [inline example/sampleuser/user.go]: We are missing this change and we are good. If you run `make test` you should see an error caused by this I believe.

Comment (ethervoid) [reply example/sampleuser/user.go]: Sorry haste makes waster :\

**Pull request merged**